src/utils/ensureDomProps.js

```diff
@@ -0,0 +1,84 @@
+import DOMProperty from 'react/lib/DOMProperty';
```

Comment:
This is unsupported and will break very soon.
Please don’t use internal React modules.
Anything inside `react/lib` is internal.